src/styles/components/migrationmonitor.less

```diff
@@ -13,19 +13,28 @@
 * limitations under the License.
 **/
 
```

Comment:
it is really odd that git thinks this is a rename and not a delete / add

Reply:
At the base level, git considers it a delete and an add. But for presentation to users, a git and add are presented a rename if the files are similar; sometimes (like this, apparently), they are above the similarity threshold when they are really different files. The git cli has options to control this presentation behavior, but you don't get that control in github

Reply:
indeed the fault of github